### PR TITLE
add package for daemonize tool

### PR DESCRIPTION
**Pull request by risicle**

### CODE REVIEW

File: pkgs/tools/system/daemonize/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+stdenv.mkDerivation rec {
+  name    = "daemonize-${version}";
+  version = "1.7.5";
+
+  src = fetchurl {
+    url    = "https://github.com/bmc/daemonize/archive/release-${version}.tar.gz";
+    sha256 = "616220b8dc5721d93bd45e63b2617dbe07cd10a572da505405d64f640b98a06b";
+  };
+
+  meta = with stdenv.lib; {
```
Comment (jagajaga) [inline pkgs/tools/system/daemonize/default.nix]: Platforms?

Comment (risicle) [inline pkgs/tools/system/daemonize/default.nix]: I can only test it on linux but according to the author supported platforms are

>  *  FreeBSD 4.x, 8.0-RELEASE, 8.1-RELEASE and 8.2-RELEASE
>  *  Red Hat Enterprise Linux 4 / CentOS 4
>  *  Solaris (SunOS 5.8, 5.10)
>  *  Fedora Core 5
>  *  Ubuntu 8, 9, 10, 11 and 12
>  *  Mac OS X 10.4 (Tiger) and 10.6 through 10.8.

Not sure what I should do...

Comment (risicle) [inline pkgs/tools/system/daemonize/default.nix]: (mark all those platforms & hope for the best? what platform(s) does travis test?)

Comment (jagajaga) [inline pkgs/tools/system/daemonize/default.nix]: Set platforms.all?

**Comment by jagajaga** [comment]
Please, set "platforms.all" :)
And squash commits?

**Comment by risicle** [comment]
Really? Upstream homepage doesn't say it runs on openbsd, and I'm fairly certain it won't work on cygwin.

**Comment by risicle** [comment]
(or netbsd)

**Comment by jagajaga** [comment]
My fault :)
So just squash commit.

File: pkgs/tools/system/daemonize/default.nix [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    description = "Runs a command as a Unix daemon";
+    homepage    = http://software.clapper.org/daemonize/;
+    license     = licenses.bsd3;
+    platforms = platforms.linux ++ platforms.freebsd ++ platforms.darwin;
```
Comment (jagajaga) [inline pkgs/tools/system/daemonize/default.nix]: And better change to `platforms = with platforms; [linux freebsd darwin];`

**Comment by risicle** [comment]
Damnit hold on

**Comment by risicle** [comment]
(Are we really allowed to put lists-of-lists in meta.platforms?)

**Comment by risicle** [comment]
Ok, also changed source hash to base32.

**Pull request merged**